### PR TITLE
Known issues

### PR DESCRIPTION
**Pull request by zengin**
- Remove isFavoriteByDefault from V1 known issues as the property is now in V1.
- Add the new failure with PutAsync due to SDK's missing feature.

### CODE REVIEW

**Review by MIchaelMainer — APPROVED** [review]

**Review by baywet — APPROVED** [review]

**Comment by baywet** [comment]
@zengin QQ: shouldn't the test be moved to the java language instead of being removed completely since they [are still failing](https://github.com/microsoftgraph/msgraph-sdk-raptor/pull/131/checks?check_run_id=1522275060)?

**Comment by baywet** [comment]
@zengin since the tests [are still failing](https://github.com/microsoftgraph/msgraph-sdk-raptor/pull/131/checks?check_run_id=1522275060) for java, shouldn't we move those exceptions in the java block instead of removing those?

**Comment by zengin** [comment]
@baywet ideally yes. Is the root cause still the same though? I don't have the local Java setup, I was hoping Java failures to go away as well because this was just a property added to V1, so I was relying on the Azure DevOps run.

I will have a quick look at the failure in the .trx because Azure DevOps is truncating error message.

**Comment by zengin** [comment]
@baywet It looks like the property is added to the Java service library 21 days ago. Why does Raptor not picking up the changes?

https://github.com/microsoftgraph/msgraph-sdk-java/blame/d12ac2c598419566504032862cdf58ca1066f65a/src/main/java/com/microsoft/graph/models/extensions/Channel.java#L57

**Comment by baywet** [comment]
@zengin after posting the message I spent a few minutes looking into that. I doesn't look like we're passing a specific version to the build process, which as far as I can remember means gradle will pull the latest version. Now it's possible it's not behaving like anticipated, running the test locally and running `gradle dependencies` in the locally generated project should tell you which version it's pulling.

**Comment by zengin** [comment]
installing gradle on my WSL instance.

**Review by MIchaelMainer — APPROVED** [review]

File: azure-pipelines/java-v1-compile-tests-known-issues.yml [hunk 1]
```diff
@@ -20,10 +20,12 @@ variables:
 steps:
 - template: java-templates/java-prepare.yml
 - template: common-templates/checkout.yml
+- template: java-templates/java-set-v1-version.yml
 - template: java-templates/java-replace-local-path.yml
   parameters:
     projectFileName: JavaV1KnownFailureTests
     javaPreviewLibPath: ''
+    javaLibVersion: $serviceLibraryVersion
```
Comment (baywet) [inline azure-pipelines/java-v1-compile-tests-known-issues.yml:28]: Wouldn't this need parenthesis?

**Review by MIchaelMainer — APPROVED** [review]

**Review by nikithauc — APPROVED** [review]

**Comment by zengin** [comment]
@baywet FYI, verified that the Java tests are passing as well after referencing the latest service library.

**Comment by baywet** [comment]
thanks for taking the time to fix that and sorry I forgot about the default value :)

**Pull request merged**